proposals/0020/README.md

```diff
@@ -0,0 +1,151 @@
+---
+id: 0020
+title: Template Versioning
+status: published
+authors: Aman Parauliya <aman@infracloud.io>
+---
+
+## Summary
+
+As per the proposal [0013], we are planning to induce workflow service as a part of the template which is used to create a workflow. As per the future perspective we may again have the requirement of modifying the existing format of template and in that case we should also provide the support of previous/current format of the template in parallel to the modification of template format/schema.  
+This is to provide backward compatibility once [0013] is implemented.
+
+## Goals and not Goals
+
+### Goals
+
+Current format or schema of the template should be compatible as well as with the new one.
+
+### Not Goals
+
+Current template schema should be depricated.
+
+## Content
+
+This feature will make the life of existing user a bit easier because he/she doesn't need to migrate to new template schema since we have the support of existing and user will also have the flexibility of using the kind of template he/she would like to use or in another words whichever template schema is fit for his/her usecase.
```

Comment:
change 
> support of existing

to 

> support for existing templates


The sentence is way too long. Please break it to smaller ones.

Reply:
Done.